contracts/cw4-group/src/msg.rs

```diff
@@ -22,6 +22,8 @@ pub enum ExecuteMsg {
     UpdateMembers {
         remove: Vec<String>,
         add: Vec<Member>,
+        // This could arguably just be a string, as it is unwrapped to default "" if None
+        code_hash: Option<String>,
```

Comment:
ah but now that i understand what the code hash is for i think this should be a required field, otherwise it would error when calling the contract in the logic right?